app/models/lettings_log.rb

```diff
@@ -687,7 +687,7 @@ def duplicates
   end
 
   def address_search_given?
-    address_line1_input.present? && postcode_full.present?
+    address_line1_input.present? && postcode_full_input.present?
```

Comment:
I'm assume this was a typo, cause postcode_full doesn't make sense for the method name and the where it's being used